cluster/juju/prereqs/ubuntu-juju.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Copyright 2014 Canonical LTD. All rights reserved.
```

Comment:
Brilliant. Thanks for following up on this Eric. I'll get the headers updated and link in the hooks.
Sent using Boxer
On Mar 26, 2015 1:50 PM, Eric Tune <notifications@github.com> wrote:In cluster/juju/prereqs/ubuntu-juju.sh:
> @@ -0,0 +1,48 @@
> +#!/bin/bash
> +
> +set -euo pipefail
> +
> +# Copyright 2014 Canonical LTD. All rights reserved.

I don't think we can take code that has anything other than the Kubernetes boilerplate at the top.  See comment on conversation tab about github hooks.

—Reply to this email directly or view it on GitHub.